### PR TITLE
[WIP] Configuration reference

### LINKED ISSUE #3458: Configuration reference
Codeception has a lot of configuration settings, but they are documented in many different places -  http://codeception.com/docs/reference/Configuration, http://codeception.com/docs/08-Customization, http://codeception.com/docs/04-FunctionalTests#Error-Reporting and so on.

I think that all possible settings should be documented in http://codeception.com/docs/reference/Configuration

### PR DESCRIPTION
**Pull request by DavertMik**
Closes #3458

### CODE REVIEW

File: docs/reference/Configuration.md [hunk 1]
```diff
@@ -1,13 +1,19 @@
 # Configuration
 
-Default your global configuration file will be this:
+## Global Configuration
 
-```yaml
-# can be changed while bootstrapping project
-actor: Tester 
+Configuration file `codeception.yml` is generate by `codecept bootstrap` command. It has preconfigured settings you can change.
```
Comment (samdark) [inline docs/reference/Configuration.md]: generate*d*

File: docs/reference/Configuration.md [hunk 2]
```diff
@@ -42,15 +51,23 @@ settings:
...
     # We set a high limit for them by default.
     memory_limit: 1024M
     
     # This value controls whether PHPUnit attempts to backup global variables
     # See https://phpunit.de/manual/current/en/appendixes.annotations.html#appendixes.annotations.backupGlobals
     backup_globals: true
+```
 
-# Global modules configuration.    
+* `modules`: allows to create share modules configuration for all included suites. However, module is can not be enabled from global config. By default sample configuration for Db module is included.
```
Comment (samdark) [inline docs/reference/Configuration.md]: share*d*

Comment (samdark) [inline docs/reference/Configuration.md]: module is can not be

"Is" should be removed.

Comment (DavertMik) [inline docs/reference/Configuration.md]: Thanks. Fixed

**Pull request merged**